samples/frameworks/react/ts/src/hooks/useTodos.ts

```diff
@@ -0,0 +1,82 @@
+/*
+  Copyright (c) Microsoft Corporation.
+  Licensed under the MIT license.
+
+  The central hook that components will integrate with and 
+  manages both the state of Todos and integration with various
+  Azure services such as ServiceBus and EventHubs.
+*/
+
+import { EventData } from "@azure/event-hubs";
+import { useState } from "react";
+import { useEventHubs } from "./useEventHubs";
+import { useServiceBus } from "./useServiceBus";
+import { ServiceBusMessage } from "@azure/service-bus";
+import { v4 as uuid } from "uuid";
+
+export interface Todo {
+  done: boolean;
+  id: string;
+  label: string;
+  noteFileName?: string;
+}
+
+export const useTodos: () => [
+  Todo[],
```

Comment:
Same question as https://github.com/Azure/azure-sdk-for-js/pull/12924/files#r544678261